### PR TITLE
[Documentation] Removing inconsistencies in botskills documentation

### PR DESCRIPTION
**Pull request by DiegoCardozo94**
Fix 3489

### Purpose
*What is the context of this pull request? Why is it being done?*
There were some inconsistences between documentation and the argument's explanation of botskills.
### Changes
*Are there any changes that need to be called out as significant or particularly difficult to grasp? (Include illustrative screenshots for context if applicable.)*
We fixed the inconsistencies in the documentation and updated some outdated docs.
### Tests
*Is this covered by existing tests or new ones? If no, why not?*

### Feature Plan
*Are there any remaining steps or dependencies before this issue can be fully resolved? If so, describe and link to any relevant pull requests or issues.*

#### General
- [ ] I have commented my code, particularly in hard-to-understand areas	
- [ ] I have added or updated the appropriate tests	
- [X] I have updated related documentation

### CODE REVIEW

File: docs/_docs/skills/handbook/add-skills-to-a-virtual-assistant.md [hunk 1]
```diff
@@ -50,8 +51,9 @@ Run the following command to add each Skill to your Virtual Assistant. This assu
...
+The `--luisFolder` parameter can be used to point the Skill CLI at the source LU files for trigger utterances (defaults to `deployment/resources/skills/` inside your assistant folder). The CLI will automatically traverse locale folder hierarchies. Also, you have to specify the `--cs` (for C#) or `--ts` (for TypeScript) argument for determining the coding language of your assistant, since each language takes different folder structures that need to be taken into consideration.
 
+> Note: The Skills can be found in the [botframework-skills](https://github.com/microsoft/botframework-skills) repository to connect the Skill to the Virtual Assistant using the botskills tool.
```
Comment (Batta32) [inline docs/_docs/skills/handbook/add-skills-to-a-virtual-assistant.md:56]: Just to simplify:
```suggestion
> Note: The Skills can be found in the [botframework-skills](https://github.com/microsoft/botframework-skills) repository.
```

File: docs/_docs/skills/tutorials/add-prebuilt-skill/2-download-and-install.md [hunk 2]
```diff
@@ -23,6 +23,8 @@ order: 2
    npm install -g botdispatch @microsoft/botframework-cli
    ```
 
+1. [.NET Core runtime](https://dotnet.microsoft.com/download/dotnet-core/2.1#runtime-2.1.0): 2.1.0 - 3.0.0
```
Comment (Batta32) [inline docs/_docs/skills/tutorials/add-prebuilt-skill/2-download-and-install.md:26]: Following [this](https://nodesource.com/blog/semver-a-primer/#:~:text=Semver%20is%20a%20specification%20outlining,graphical%20user%20interface%20(GUI).) and knowing that the previous commit was `2.1.0 or higher but lower than 3.0.0` which means `>=2.1.0 <3.0.0`. The `2.1.0 - 3.0.0` is not the same as before.

This should be changed to `^2.1.0`

@LCarbonellRod Can you confirm this?

File: docs/_docs/skills/tutorials/create-skill/csharp/2-download-and-install.md [hunk 3]
```diff
@@ -22,6 +22,7 @@ order: 2
    ```
    npm install -g botdispatch @microsoft/botframework-cli
    ```
+1. [.NET Core runtime](https://dotnet.microsoft.com/download/dotnet-core/2.1#runtime-2.1.0): 2.1.0 - 3.0.0
```
Comment (Batta32) [inline docs/_docs/skills/tutorials/create-skill/csharp/2-download-and-install.md:25]: Change the semver

File: tools/botskills/README.md [hunk 4]
```diff
@@ -5,9 +5,11 @@ The Botskills Tool is a command line tool to manage the skills connected to your
 - [Node.js](https://nodejs.org/) version 10.8 or higher
 - Install [@microsoft/botframework-cli](https://www.npmjs.com/package/@microsoft/botframework-cli)
 
+
 ```shell
 npm install -g botdispatch @microsoft/botframework-cli
 ```
+- [.NET Core runtime](https://dotnet.microsoft.com/download/dotnet-core/2.1#runtime-2.1.0): 2.1.0 - 3.0.0
```
Comment (Batta32) [inline tools/botskills/README.md:12]: Change the semver

File: docs/_docs/skills/handbook/add-skills-to-a-virtual-assistant.md [hunk 5]
```diff
@@ -19,6 +19,7 @@ toc: true
     ```shell
     npm install -g botdispatch @microsoft/botframework-cli
     ```
+- [.NET Core runtime](https://dotnet.microsoft.com/download/dotnet-core/2.1#runtime-2.1.0): 2.1.0 - 3.0.0
```
Comment (Batta32) [inline docs/_docs/skills/handbook/add-skills-to-a-virtual-assistant.md:22]: Change the semver

File: docs/_docs/skills/handbook/add-skills-to-a-virtual-assistant.md [hunk 6]
```diff
@@ -92,7 +95,9 @@ Run the following command to update a Skill to your Virtual Assistant. This assu
...
+The `--luisFolder` parameter can be used to point the Skill CLI at the source LU files for trigger utterances. For Skills provided within this repo these can be found in the `Deployment/Resources/LU` folder of each Skill. The CLI will automatically traverse locale folder hierarchies. Also, you have to specify the `--cs` (for C#) or `--ts` (for TypeScript) argument for determining the coding language of your assistant, since each language takes different folder structures that need to be taken into consideration.
+
+> Note: The Skills can be found in the [botframework-skills](https://github.com/microsoft/botframework-skills) repository to connect the Skill to the Virtual Assistant using the botskills tool.
```
Comment (Batta32) [inline docs/_docs/skills/handbook/add-skills-to-a-virtual-assistant.md:100]: Just to simplify:
```suggestion
> Note: The Skills can be found in the [botframework-skills](https://github.com/microsoft/botframework-skills) repository.
```

File: docs/_docs/skills/tutorials/add-prebuilt-skill/2-download-and-install.md (hunk 2 above)
Comment (LCarbonellRod) [reply docs/_docs/skills/tutorials/add-prebuilt-skill/2-download-and-install.md:26]: Yes! it should be `^2.1.0`.

Comment (DiegoCardozo94) [reply docs/_docs/skills/tutorials/add-prebuilt-skill/2-download-and-install.md:26]: We simplify this `>=2.1.0 <3.0.0` to `^2.1.0`?

Comment (Batta32) [reply docs/_docs/skills/tutorials/add-prebuilt-skill/2-download-and-install.md:26]: `^2.1.0` means `>=2.1.0 <3.0.0`, so yes

**Comment by DiegoCardozo94** [comment]
> LGTM! @LCarbonellRod will review it and as soon as you have their confirmation you can push them to the PR.
> 
> Just minor question, have you double checked all the location that the .NET Core runtime is used in the docs? Just to update the semver in every doc

Yes @Batta32, I checked and updated the semver version in every doc. I detected five documents that contains this note.

**Comment by Batta32** [comment]
Can we close this PR? @DiegoCardozo94

**Pull request closed without merging**